File: Conditionals/old_enough.py
# Tatiana Susov Old Enough Py

print("Hello, welcome to my program.\n This program will tell you if you are old enough to have certain oppotunities. ")

name = input("What is your name?\n")

print("Hi,", name)

num = int(input("Please enter your current age.\n"))
if num >= 4 and num <= 14:
    print("You are old enough to attend school!")
elif num == 15:
    print("You are old enough to get a learners permit!")
elif num >= 16 and num <= 18:
    print("You are old enough to drive!")
elif num >= 18:
    print("You are old enough to vote!")
else:
    print("You are not old enough to have these certain oppotunities.")

File: Conditionals/old_enough.c
//Tatiana Susov, Old Enough C Practice
#include <stdio.h>

int num;

int main(void){
    
    printf("Hello, welcome to my program.\n This program will tell you if you are old enough to have certain opportunities.\n");
    printf("%d How old are you ?\n", num);
    scanf("%d", &num);
    printf("You are %d.\n", num);
    if(num > 3 && num < 15 ){
        printf("You are old enough to attend school!\n");
    }else if(num == 15){
        printf("You are old enough to get a learners permit!\n");
    }else if(num == 16 || num == 17){
        printf("You are old enough to drive!\n");
    }else if(num > 17){
        printf("You are old enough to vote!\n");
    }else{
        printf("You are not old enough to have these certain opportunities.");
    }
    return 0;
}